django_project/users/views.py

```diff
@@ -1,10 +1,27 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from .forms import UserRegisterForm
 
 
 def register(request):
-    return render(request, 'users/register.html', {'title': 'Register'})
+    if request.method == 'POST':
+        form = UserRegisterForm(request.POST)
+        form.fields['username'].label = 'Email'
+        if form.is_valid():
+            form.save()
+            username = form.cleaned_data.get('username')
+            messages.success(request, f'Account Created for {username}! You are now able to log in.')
+            return redirect('login')
+    else:
+        form = UserRegisterForm()
+    return render(request, 'users/register.html', {'form': form})
 
 
 def login(request):
     return render(request, 'users/login.html', {'title': 'Login'})
```

Comment:
Please correct me if I'm wrong, but wouldn't we need to handle POST requests here? It looks like your login.html page contains a form that submits post requests to this view.

Reply:
I think that because I'm not storing any information to the database, handling it isn't needed. Either that or Django handles it.

Reply:
Ok gotcha, makes sense.